extension/chrome/settings/index.ts

```diff
@@ -309,6 +311,38 @@ Catch.try(async () => {
     }));
   };
 
+  const renderBlogPostList = async () => {
+    const errorRetrievingPosts = () => {
+      Xss.sanitizeAppend('.blog_post_list', `<div class="line>Error retrieving blog posts...try again later.</div>\n`);
+    };
+    try {
+      const res = await Api.ajax({
```

Comment:
this should be as a method somewhere in the `Api` class. Also, we typically put short object literals on one line eg `{url: 'https://flowcrypt.com/feed', method: 'GET', dataType: 'json', async: true}`. You can probably omit the method and async (I think), so that would be `{url: 'https://flowcrypt.com/feed',  dataType: 'json'}`.